flontend/src/components/HelloWorld.vue

```diff
@@ -0,0 +1,64 @@
+<template>
+  <div id="welcome">
+    <h1>5W1HGameSite</h1>
+
+    <section v-if="errored">
+      <p>Error</p>
+    </section>
+
+    <section v-else>
+      <div class="axios_data">
+        {{ info }}:
+      </div>
+
+    </section>
+  </div>
+</template>
+
+<script>
+import axios from 'axios'
+
+export default {
+  name: 'axios_data',
+  data () {
+    return {
+      info: null,
```

Comment:
期待される型が配列かオブジェクトだから、初期値はnullではなく
```suggestion
      info: [],
```
もしくは
```suggestion
      info: {},
```
にした方がいいかも